src/OpenTelemetry.Instrumentation.AspNetCore/Implementation/HttpInListener.cs

```diff
@@ -263,12 +269,8 @@ public void OnStopActivity(Activity activity, object payload)
             }
         }
 
-#if NET7_0_OR_GREATER
         var tagValue = activity.GetTagValue("IsCreatedByInstrumentation");
         if (ReferenceEquals(tagValue, bool.TrueString))
-#else
```

Comment:
@Kielek we would need to keep this.

Reply:
0c1af8ba03aefac8547bfdb4c6d0a8ad541d4bc3